packages/manager/modules/octavia-load-balancer/src/index.js

```diff
@@ -12,7 +12,7 @@ angular
   .config(
     /* @ngInject */ ($stateProvider) => {
       $stateProvider.state('octavia-load-balancer.**', {
-        url: '/octavia-load-balancer',
+        url: '/octavia-load-balancer/{projectId:[0-9a-zA-Z]{32}}',
```

Comment:
```suggestion
        url: '{projectId:[0-9a-zA-Z]{32}}/octavia-load-balancer',
```

Reply:
Because it is a independent app, we made a meeting with core team and agreed to use url like this /manager/#/octavia-load-balancer/projectId. Because it is weird to have /manager/#/projectId/octavia-load-balancer